pkg/chains/formats/intotoite6/util/utils.go

```diff
@@ -0,0 +1,196 @@
+/*
+Copyright 2020 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"sort"
+	"strings"
+
+	"github.com/google/go-containerregistry/pkg/name"
+	slsa "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/v0.2"
+	"github.com/tektoncd/chains/pkg/artifacts"
+	"github.com/tektoncd/chains/pkg/chains/objects"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"go.uber.org/zap"
+
+	intoto "github.com/in-toto/in-toto-golang/in_toto"
+)
+
+const (
+	TektonID                     = "https://tekton.dev/attestations/chains@v2"
+	TektonPipelineRunID          = "https://tekton.dev/attestations/chains/pipelinerun@v2"
+	CommitParam                  = "CHAINS-GIT_COMMIT"
+	UrlParam                     = "CHAINS-GIT_URL"
```

Comment:
```suggestion
	URLParam                     = "CHAINS-GIT_URL"
```

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
fixed